src/viur/core/modules/file.py

```diff
@@ -449,6 +493,53 @@ def hmac_sign(data: t.Any) -> str:
     def hmac_verify(data: t.Any, signature: str) -> bool:
         return hmac.compare_digest(File.hmac_sign(data.encode("ASCII")), signature)
 
+    @staticmethod
+    def create_serve_parameters(serving_url: str) -> t.Iterable[str] | None:
+        """
+        Splits a serving URL into its components, used by serve function.
+        :param serving_url: the serving URL to be split
+        :return: return a tuple of host and key
+        """
+        regex_result = re.match(
+            r"^https:\/\/(.*?)\.googleusercontent\.com\/(.*?)$",
+            serving_url
+        )
+
+        if regex_result:
+            return regex_result.groups()
+        return regex_result
+
+    @staticmethod
+    def create_serve_url(
+        serving_url: str,
+        size: t.Optional[int] = None,
+        filename: t.Optional[str] = None,
+        options: str = "",
+        download: bool = False
+    ) -> str:
+
+        serve_url = f"{File.SERVE_URL_PREFIX}"
```

Comment:
It's already a string

```suggestion
        serve_url = File.SERVE_URL_PREFIX
```